dna/course/zomes/courses/code/src/content/entry.rs

```diff
@@ -0,0 +1,58 @@
+use hdk::{
+    entry_definition::ValidatingEntryType, 
+    holochain_core_types::{dna::entry_types::Sharing, validation::EntryValidationData},
+    holochain_json_api::{error::JsonError, json::JsonString},
+    holochain_persistence_api::cas::content::Address, 
+};
+use holochain_entry_utils::HolochainEntry; 
+
+#[derive(Serialize, Deserialize, Debug, DefaultJson, Clone)]
+pub struct Content {
+    pub title: String, 
+    pub timestamp: u64, 
+    pub anchor_address: Address, 
```

Comment:
do you mean `section_anchor_address` here? or is it something different?

Reply:
I just copied it from the course entry and didn't take the time to think it through. Thanks for your explanation during homework review.